examples/github-stars-example/README.md

```diff
@@ -0,0 +1,307 @@
+
+1. Clone the repo
+```
+git clone https://github.com/vmware/versatile-data-kit.git
+```
+2. Install vdk
+```
+pip install -U pip setuptools wheel
+pip install quickstart-vdk
+```
+3. Now do vdk create, enter the job name, team and location
+```
+vdk create
+
+Job Name: local-github-stars
+Job Team: test
+Path to where sample data job will be created locally [/home/gita/vdk]:
+Data Job with name local-github-stars created locally in /home/gita/vdk/local-github-stars.
+```
+4. Clear the new directory and copy example files to it
+```
+rm local-github-stars/*
+cp versatile-data-kit/examples/github-stars-example/github-stars-example-job/* local-github-stars/
+```
+5. Add your GitHub token and repo name to the file 00_properties.py
+```
+vim local-github-stars/00_properties.py
+```
+NB! Repo Path is only the username and reponame not the full link "repo_path": "vmware/versatile-data-kit"
+
+6. Install requirements
+```
+pip install -r local-github-stars/requirements.txt
+```
+7. Run the job
+```
+vdk run local-github-stars
+```
+8. Now you can see your data by running:
+```
+vdk sqlite-query -q "Select * from github_star_history"
+```
+9. To create the dashboard, run:
+```
+streamlit run local-github-stars/40_build_streamlit_dashboard.py
+```
+Your webapp should be up, here: http://localhost:8501 🥳
+
+To change the streamlit colors you can modify the file:
+```
+vim ~/.streamlit/config.toml
+```
+
+Code
+----
+
+The relevant Data Job and Airflow DAG code is available
+[here](https://github.com/vmware/versatile-data-kit/tree/main/examples/github-stars-example).
+
+The code can be reused to track Hithub Star History for any GitHub repo.
+
+Data
+--------
+
+We get the data from GitHub API using PyGithub library.
+To display the history we need to get the date when user starred the repository and count of the stars.
+
+Requirements
+------------
+
+To run this example, you need
+* Versatile Data Kit
+* PyGithub
+* Sqlite
+* Streamlit
+* Pandas
+* Plotly
+* Versatile Data Kit Properties FS plugin
+
+Configuration
+-------------
+
+If you have not done so already, you can install Versatile Data Kit and the
+plugins required for this example by running the following commands from a terminal:
+```console
+pip install quickstart-vdk
+pip install vdk-properties-fs
+```
+Note that Versatile Data Kit requires Python 3.7+. See the
+[Installation page](https://github.com/vmware/versatile-data-kit/wiki/Installation#install-sdk) for more details.
+
+In the config.ini file following lines are necessary to enable the plugin and add Sqlite configuration
+
+```
+db_default_type=SQLITE
+ingest_method_default=sqlite
+properties_default_type=fs-properties-client
+```
+
+Data Job
+--------
+
+Data job has the following structure:
+
+```
+github-stars-example-job/
+├── 00_properties.py
+├── 10_drop_table.sql
+├── 20_create_table.sql
+├── 30_stargazers.py
+├── 40_build_streamlit_dashboard.py
+├── config.ini
+├── requirements.txt
+```
+
+<details>
```

Comment:
Are those details needed ? They are just repeating the source code but now we need to keep then in sync ... ?